src/content/structured/components/badge/code.mdx

```diff
@@ -0,0 +1,755 @@
+    snippet: `<div style="padding:16px;">
+  <ic-chip label="Small" style="margin-right:16px;">
+    <ic-badge
+      text-label="1"
+      slot="badge"
+      position="near"
+      size="small"
+      variant="info"
+    >
+    </ic-badge>
+  </ic-chip>
+  <ic-chip label="Default" style="margin-right:16px;">
+    <ic-badge text-label="1" slot="badge" position="near" variant="info">
+    </ic-badge>
+  </ic-chip>
+  <ic-chip label="Large">
+    <ic-badge
+      text-label="1"
+      slot="badge"
+      position="near"
+      size="large"
+      variant="info"
+    >
+    </ic-badge>
+  </ic-chip>
+</div>`,
+  },
+  {
+    language: "React",
+    snippet: `<div style={{padding:"16px"}}>
+  <IcChip label="Small" style={{marginRight:"16px"}}>
+    <IcBadge
+      text-label="1"
+      slot="badge"
+      position="near"
+      size="small"
+      variant="info"
+    >
+    </IcBadge>
+  </IcChip>
+  <IcChip label="Default" style={{marginRight:"16px"}}>
+    <IcBadge
+      text-label="1"
+      slot="badge"
+      position="near"
+      variant="info"
+    >
+    </IcBadge>
+  </IcChip>
+  <IcChip label="Large">
+    <IcBadge
+      text-label="1"
+      slot="badge"
+      position="near"
+      size="large"
+      variant="info"
+    >
+    </IcBadge>
+  </IcChip>
+</div>`,
+  },
+];
+
+<ComponentPreview snippets={snippetsNear}>
+  <div  style={{ display: "grid" }} >
+    <div style={{padding:"16px"}}>
+      <IcChip label="Small" style={{marginRight:"16px"}}>
+        <IcBadge
+          text-label="1"
+          slot="badge"
+          position="near"
+          size="small"
+          variant="info"
+         />
+      </IcChip>
+      <IcChip label="Default" style={{marginRight:"16px"}}>
+        <IcBadge
+          text-label="1"
+          slot="badge"
+          position="near"
+          variant="info"
+         />
+      </IcChip>
+      <IcChip label="Large">
+        <IcBadge
+          text-label="1"
+          slot="badge"
+          position="near"
+          size="large"
+          variant="info"
+         />
+      </IcChip>
+    </div>
+  </div>
+</ComponentPreview>
+
+### Position inline
+
+export const snippetsInline = [
+  {
+    language: "Web component",
+    snippet: `<ic-tab-context>
+  <ic-tab-group label="Example tab group">
+    <ic-tab
+      >Small<ic-badge
+        text-label="1"
+        slot="badge"
+        position="inline"
+        size="small"
+        accessible-label="1 notification found"
+        variant="info"
+      ></ic-badge
+    ></ic-tab>
+    <ic-tab
+      >Default<ic-badge
+        text-label="1"
+        slot="badge"
+        position="inline"
+        accessible-label="1 notification found"
+        variant="info"
+      ></ic-badge
+    ></ic-tab>
+    <ic-tab
+      >Large<ic-badge
+        text-label="1"
+        slot="badge"
+        position="inline"
+        size="large"
+        accessible-label="1 notification found"
+        variant="info"
+      ></ic-badge
+    ></ic-tab>
```

Comment:
These code examples just need the closing tags tidying up a bit. Same changes need to be applied to the react code example too
```suggestion
    <ic-tab>
      Small
      <ic-badge
        text-label="1"
        slot="badge"
        position="inline"
        size="small"
        accessible-label="1 notification found"
        variant="info"
      >
      </ic-badge>
    </ic-tab>
    <ic-tab>
      Default
      <ic-badge
        text-label="1"
        slot="badge"
        position="inline"
        accessible-label="1 notification found"
        variant="info"
      >
      </ic-badge>
    </ic-tab>
    <ic-tab>
      Large
      <ic-badge
        text-label="1"
        slot="badge"
        position="inline"
        size="large"
        accessible-label="1 notification found"
        variant="info"
      >
      </ic-badge>
    </ic-tab>
```